python/cuml/ensemble/randomforest_shared.pxd

```diff
@@ -51,7 +51,7 @@ cdef extern from "cuml/tree/decisiontree.hpp" namespace "ML::DecisionTree":
         float max_features
         int n_bins
         int split_algo
-        int min_rows_per_node
+        int min_samples_leaf
```

Comment:
this should also contain `min_samples_split` no?

Reply:
Ah that's right, I'll fix it.